src/components/delete-track-box.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+    Box,
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    FormControl,
+    FormHelperText,
+    TextField,
+} from "@material-ui/core";
+import { Track } from "../store/types";
+
+export interface DeleteTrackBoxProps {
+    isOpen: boolean;
+    onClose: () => void;
+    track: Track;
+    onSubmit: () => void;
+}
+
+export default function DeleteTrackBox({
```

Comment:
let's rename this to `DeleteTrackDialog`, to better describe what sort of component this is